adoc/extensions/sycl_khr_free_function_commands.adoc

```diff
@@ -0,0 +1,928 @@
+
+template <typename KernelType>
+void launch(sycl::queue q, sycl::range<2> r, const KernelType& k);    (5)
+
+template <typename KernelType>
+void launch(sycl::queue q, sycl::range<3> r, const KernelType& k);    (6)
+
+}
+----
+_Effects (1-3)_: Equivalent to [code]#h.parallel_for(r, k)#.
+
+_Effects (4-6)_: Equivalent to [code]#q.submit([&](handler& h) { launch(h, r,
+k); })#.
+
+'''
+
+.[apititle]#launch# (kernel object)
+[source,role=synopsis,id=api:launch-kernel]
+----
+namespace sycl::khr {
+
+template <typename... Args>
+void launch(sycl::handler& h, sycl::range<1> r,
+            const sycl::kernel& k, Args&&... args); (1)
+
+template <typename... Args>
+void launch(sycl::handler& h, sycl::range<2> r,
+            const sycl::kernel& k, Args&&... args); (2)
+
+template <typename... Args>
+void launch(sycl::handler& h, sycl::range<3> r,
+            const sycl::kernel& k, Args&&... args); (3)
+
+template <typename... Args>
+void launch(sycl::queue q, sycl::range<1> r,
+            const sycl::kernel& k, Args&&... args); (4)
+
+template <typename... Args>
+void launch(sycl::queue q, sycl::range<2> r,
+            const sycl::kernel& k, Args&&... args); (5)
+
+template <typename... Args>
+void launch(sycl::queue q, sycl::range<3> r,
+            const sycl::kernel& k, Args&&... args); (6)
+
+}
+----
+_Effects_: Enqueues a kernel object that is invoked for every work-item in the
+specified [code]#sycl::range#.
+The arguments in [code]#args# are passed to the kernel in the same order.
+
+'''
+
+.[apititle]#launch_reduce# (kernel function)
+[source,role=synopsis,id=api:launch_reduce]
+----
+namespace sycl::khr {
+
+template <typename KernelType, typename... Reductions>
+void launch_reduce(sycl::handler& h, sycl::range<1> r,
+                   const KernelType& k, Reductions&&... reductions); (1)
+
+template <typename KernelType, typename... Reductions>
+void launch_reduce(sycl::handler& h, sycl::range<2> r,
+                   const KernelType& k, Reductions&&... reductions); (2)
+
+template <typename KernelType, typename... Reductions>
+void launch_reduce(sycl::handler& h, sycl::range<3> r,
+                   const KernelType& k, Reductions&&... reductions); (3)
+
+template <typename KernelType, typename... Reductions>
+void launch_reduce(sycl::queue q, sycl::range<1> r,
+                   const KernelType& k, Reductions&&... reductions); (4)
+
+template <typename KernelType, typename... Reductions>
+void launch_reduce(sycl::queue q, sycl::range<2> r,
+                   const KernelType& k, Reductions&&... reductions); (5)
+
+template <typename KernelType, typename... Reductions>
+void launch_reduce(sycl::queue q, sycl::range<3> r,
+                   const KernelType& k, Reductions&&... reductions); (6)
+
+}
+----
+_Constraints_: The parameter pack consists of 0 or more objects created by the
+[code]#sycl::reduction# function.
+
+_Effects (1-3)_: Equivalent to [code]#h.parallel_for(r, reductions..., k)#.
```

Comment:
```suggestion
_Effects (1-3)_: Equivalent to [code]#+h.parallel_for(r, reductions..., k)+#.
```

Minor editorial nit ... this will prevent the `...` from being rendered as an ellipsis (which we do elsewhere when `...` is part of a code snippet).  Same comment for the other places below where `...` appears in an inline `[code]` role.

Reply:
Fixed in 9747f7a4ad3deecda7a422be4ba89730c458da59.